src/main/java/by/a1qa/utils/RandomNumberGenerator.java

```diff
@@ -0,0 +1,17 @@
+package by.a1qa.utils;
+
+public class RandomNumberGenerator {
+    private static int randomNumber;
+
+    public RandomNumberGenerator(int border) {
+        setRandomNumber(border);
+    }
+
+    public static int getRandomNumber() {
+        return randomNumber;
+    }
+
+    private static void setRandomNumber(int border) {
+        randomNumber = 0 + (int) (Math.random() * border);
+    }
+}
```

Comment:
this class is a mess: why do we have static and not static methods?
replace this all with
```java
public class NumberUtils {

    private NumberUtils() {}

    public static int getRandomNumber(int maxExcluded) {
        return (int) (Math.random() * maxExcluded);
    }
}